CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## 5.0.0-beta1 (Sep 21, 2016)
  - Migrated Logstash's internal logging framework to Log4j2. This enhancement provides the following features:
    - Support changing logging level dynamically at runtime through REST endpoints. New APIs have been exposed 
-     under `_node/settings` to update log levels. Also a new endpoint `_node/logging` was added to return all 
+     under `_node/logging` to update log levels. Also a new endpoint `_node/logging` was added to return all 
```

Comment:
I see `"path":"/_node/settings","status":404,"error":{"message":"Not Found"}}` when I use `_node/settings`. But when I use `_node/logging` I see `{"host":"Rhodas-MacBook-Pro.local","version":"5.0.0-beta1","http_address":"127.0.0.1:9600","acknowledged":true}` 

I'm using this curl command:
`````
curl -XPUT 'localhost:9600/_node/logging
' -d '{
    "logger.logstash.outputs.elasticsearch" : "INFO"
}'
```````
Am I doing something wrong?